README.md

```diff
@@ -4,9 +4,12 @@ Rust implementation of JSON-RPC 2.0 Specification.
 Transport-agnostic `core` and transport servers for `http`, `ipc`, `websockets` and `tcp`.
 
 [![Build Status][travis-image]][travis-url]
+[![Build Status][appveyor-image]][appveyor-url]
 
 [travis-image]: https://travis-ci.org/paritytech/jsonrpc.svg?branch=master
 [travis-url]: https://travis-ci.org/paritytech/jsonrpc
+[appveyor-image]: https://ci.appveyor.com/api/projects/status/github/devops-parity/jsonrpc?svg=true
+[appveyor-url]: https://ci.appveyor.com/project/devops-parity/jsonrpc/branch/master
```

Comment:
why is it `devops-parity` and not `paritytech`?

Reply:
I'm not sure `paritytech` account is set up properly on appveyor: https://ci.appveyor.com/project/paritytech/jsonrpc doesn't seem to exist, but I've changed it anyway.